feature/bgp/policybase/otg_tests/default_policies_test/bgp_default_policies_test.go

```diff
@@ -195,6 +192,11 @@ func configurePrefixMatchPolicy(t *testing.T, dut *ondatra.DUTDevice, prefixSet,
 	pset := rp.GetOrCreateDefinedSets().GetOrCreatePrefixSet(prefixSet)
 	for _, pref := range ipPrefixSet {
 		pset.GetOrCreatePrefix(pref+"/"+maskLen, prefixSubnetRange)
+		mode := oc.PrefixSet_Mode_IPV4
+		if maskLen == maskLen128 {
+			mode = oc.PrefixSet_Mode_IPV6
+		}
+		pset.SetMode(mode)
```

Comment:
There is a deviation which would be required for other vendors to skip this configuration, it needs to be incorporated here - 

https://github.com/openconfig/featureprofiles/blob/f4aec340a45833a19e7dc40095b30567a757fc5a/internal/deviations/deviations.go#L839-L842

Reply:
fixed